dream_layer_backend/dream_layer_backend_utils/shared_lora_utils.py

```
@@ -0,0 +1,181 @@
+    """
+    Create a key mapping between base model and LoRA keys.
+
+    Args:
+        base_state_dict: Base model state dict
+        lora_state_dict: LoRA state dict
+
+    Returns:
+        Dict[str, str]: Mapping from LoRA keys to base model keys
+    """
+    key_map = {}
+    for key in base_state_dict.keys():
```

Ruff (0.11.9)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)